src/components/Contact.js

```diff
@@ -0,0 +1,20 @@
+import useFetch from '../Hooks/Use-fetch';
+
+const Contact = () => {
+  const { loading } = useFetch('http://localhost:8000/courses');
+
+  return (
+    <div className="contact-us">
+      { loading && (
+      <div className="loading">
+        Fetching contacts...
+        <br />
+        Please wait
+      </div>
+      )}
+      {!loading && <h4>Contact us via the following channels:</h4>}
+    </div>
+  );
+};
+
+export default Contact;
```

Comment:
You may remove this component as it is not doing anything really. Even the content that is being fetched from the server is not even used. so just remove the COntact.js component and every reference it might have